azure-devops/run-tests.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
I would probably make most of this file part of a `cross-test.yml` file and then create a `cross-build-test.yml` file which includes both of the `cross-build.yml` and `cross-test.yml` files.